lib/project_types/script/layers/application/push_script.rb

```diff
@@ -14,24 +14,23 @@ def call(ctx:, force:, project:)
 
             extension_point = ExtensionPoints.get(type: script_project.extension_point_type)
             library_name = extension_point.libraries.for(script_project.language)&.package
-            raise Infrastructure::Errors::LanguageLibraryForAPINotFoundError.new(
-              language: script_project.language,
-              api: script_project.extension_point_type
-            ) unless library_name
-
             library = {
               language: script_project.language,
               version: task_runner.library_version(library_name),
-            }
+            } if library_name
```

Comment:
What happens on [this line](https://github.com/Shopify/shopify-cli/pull/1936/files#diff-285a9f0097add4416f72d78eae8d034ca0eef7a60eccacedfa2c1e3d2b90e0f2R23) if `library_name` is not truthy? Will it error because `library` was never defined?

Reply:
`library` will be nil in the case of `library_name` not being truthy.

```ruby
irb(main):007:1* library = {
irb(main):008:1*   a: "b",
irb(main):009:1*   c: "d",
irb(main):010:0> } if false
=> nil
irb(main):011:0> library
=> nil
```

Then the nil is pass along which is fine.